scc/building-from-source.md

```diff
@@ -449,3 +453,75 @@ Instead of a `GitRepository` object, an `ImageRepository` is created:
 `ImageRepository` provides the same semantics as `GitRepository`,
 except that it looks for source code in container image registries rather than
 Git repositories.
+
+## <a id="maven-artifact"></a>Maven Artifact
+
+This approach is intended to be used to aid integration with existing CI/CI
```

Comment:
```suggestion
This approach is intended to be used to aid integration with existing CI/CD
```

Reply:
Changed to "CI" as per @scothis 's suggestion.